ejercicio2/src/test/java/practica2/PostTest.java

```diff
@@ -0,0 +1,69 @@
+package practica2;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PostTest {
+	static Post post[] = new Post[10];
+	@Before
+	public static void init() {
+		post[0] = new Post("note 1");
+		post[1] = new Post("note 2");
+		post[2] = new Post("note 3");
+		post[3] = new Post("note 4");
+		post[4] = new Post("note 5");
+		post[5] = new Post("note 6");
+		post[6] = new Post("note 7");
+		post[7] = new Post("note 8");
+		post[8] = new Post("note 9");
+		post[9] = new Post("note 10");
+		for (int i = 0; i <= 9; i++){
```

Comment:
Because the topic say that: "When the program is run for the first time, the list is empty"